main.py

```diff
@@ -0,0 +1,14 @@
+import config
+import praw
+import instances as i
+
+reddit = praw.Reddit('prbot')
+#reddit = praw.Reddit(client_id=config.CLIENT_ID,
+	     #client_secret=config.CLIENT_SECRET,
+	     #refresh_token=config.ACCESS_TOKEN,
+	     #user_agent='testscript by /u/asdfg')
+
+subreddit1 = reddit.subreddit('Jokes')
+subreddit = reddit.subreddit('subredditname')
+
+i.get_jokes(subreddit1,subreddit)
```

Comment:
I was hoping we could have a list of functions here, get a bunch of posts and then call the functions on these posts.

Something like

```
functions = [reply_with_jokes]
while true:
    get x posts from a subreddit
    for f in functions:
        f(posts)     # each function applied on the posts performs its tasks
    sleep 5 seconds before doing it again
```

This would help later, as we'd just have to add new functions to this list and the bot would call them automatically, what do you think?